File: solutions/1347/1347-kir3i.cpp
class Solution {
public:
    int cnt[26];
    int ans;
    int minSteps(string s, string t) {
        for(const char &c: s)
            cnt[c-'a']++;
        for(const char &c: t)
            if(cnt[c-'a']-- <= 0)
                ans++;
        return ans;
    }
};

File: solutions/1347/1347-kir3i.java
class Solution {
    public int[] cnt = new int[26];
    public int ans = 0;
    public int minSteps(String s, String t) {
        for(int i=0; i<s.length(); i++)
            cnt[s.charAt(i)-'a']++;
        for(int i=0; i<t.length(); i++)
            if(cnt[t.charAt(i)-'a']-- <= 0)
                ans++;
        return ans;
    }
}
